DocC Error

```
 /Users/runner/work/ApplicationKit/ApplicationKit/Sources/ApplicationKit/LocationApplication/LocationApplication.swift:28:15: error: cannot override 'locationManager' which has been marked unavailable: Implement -locationManager:didUpdateLocations: instead
    open func locationManager(_ manager: CLLocationManager, didUpdateTo newLocation: CLLocation, from oldLocation: CLLocation) { }
```

https://github.com/0xLeif/ApplicationKit/actions/runs/7270083770/job/19808642563

--- a/Sources/ApplicationKit/LocationApplication/LocationApplication.swift
+++ b/Sources/ApplicationKit/LocationApplication/LocationApplication.swift
@@ -25,7 +25,6 @@ open class LocationApplication: LifecyleApplication, CLLocationManagerDelegate {
     open func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) { }
     open func locationManager(_ manager: CLLocationManager, didFailWithError error: Error) { }
     open func locationManager(_ manager: CLLocationManager, didRange beacons: [CLBeacon], satisfying beaconConstraint: CLBeaconIdentityConstraint) { }
-    open func locationManager(_ manager: CLLocationManager, didUpdateTo newLocation: CLLocation, from oldLocation: CLLocation) { }
     open func locationManager(_ manager: CLLocationManager, didStartMonitoringFor region: CLRegion) { }
     open func locationManager(_ manager: CLLocationManager, didDetermineState state: CLRegionState, for region: CLRegion) { }
     open func locationManager(_ manager: CLLocationManager, didFailRangingFor beaconConstraint: CLBeaconIdentityConstraint, error: Error) { }

--- a/Sources/ApplicationKit/LocationApplication/LocationManager.swift
+++ b/Sources/ApplicationKit/LocationApplication/LocationManager.swift
@@ -1,7 +1,7 @@
 import AppState
 import CoreLocation
 
-fileprivate extension Application {
+private extension Application {
     var coreLocationManager: Dependency<CLLocationManager> {
         dependency(CLLocationManager())
     }
@@ -9,7 +9,7 @@ fileprivate extension Application {
 
 /// A class that wraps around a CLLocationManager instance from AppState's Application singleton.
 open class LocationManager {
-    @AppDependency(\.coreLocationManager) private var manager: CLLocationManager
+    @AppDependency(\.coreLocationManager) public var manager: CLLocationManager
 
     /// The delegate object to receive update events.
     public var delegate: CLLocationManagerDelegate? {

